.pre-commit-config.yaml

```diff
@@ -71,7 +71,7 @@ repos:
         files: ^(homeassistant|pylint)/.+\.py$
       - id: pylint
         name: pylint
-        entry: script/run-in-env.sh pylint -j 0 --ignore-missing-annotations=y
+        entry: script/run-in-env.sh pylint -j 0
```

Comment:
This change has nothing to do with this PR. Please revert it.

Reply:
I also use dev containers and don't have this problem.
Could it be that your devcontainer is old? You should try to create a fresh dev env.